src/pages/products/seller/page.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC } from 'react';
+import { Flex, Heading, Input, Checkbox, CheckboxGroup, Stack, Text } from '@chakra-ui/react';
+import { colors } from '@/styles/theme/@colors';
+
+const ProductRegistrationPage: FC = () => {
```

Comment:
이 페이지에선 게시 정책 설정, 게시글 제목, 상품 이름, 상품 분류, 게시 마감 기한 및 상품 상세 소개와 같은 여러 섹션이 있는데, 각 섹션을 개별 컴포넌트로 분할해서 (수정) 및 가독성을 높일 수 있지 않을까요? 

ex)
```suggestion
const PostPolicySection: FC = () => {
  // 게시 정책 설정 관련 코드
}

const ProductCategorySection: FC = () => {
  // 상품 분류 관련 코드
}

```

Reply:
의견 나누는 모습 보기 좋습니다~
취향 차이긴 한데, 제시한 두 가지는 분리할만큼 코드가 방대하거나 로직이 복잡하지 않아서 전 한 컴포넌트에 그냥 나열하는게 좋다 생각해요. 예외로 아래 이미지 셀렉터는 다른페이지에서 가져다 쓸 수도 있고, 로직이 어느정도 들어가기에 별도 컴포넌트로 분리하는게 좋구요

<img width="716" alt="image" src="https://github.com/geezers-io/Stock2U-front/assets/57123802/781df75c-6092-404d-81aa-88f04f960dc2">